.travis.yml

```diff
@@ -290,7 +292,7 @@ install:
       ~/.venv-no-setuptools/bin/pip uninstall -y setuptools;
       ~/.venv-no-setuptools/bin/python -We:invalid setup.py -q install;
     fi
-    python -We:invalid -m compileall -f -q sympy/;
+    python -We::SyntaxWarning -m compileall -f -q sympy/;
```

Comment:
What is the effect of changing this? Remember that this should pick up many different types of errors.

Take a look at #16154 which was the last time someone edited this line. There was some discussion there about how to know that the line was still achieving it's desired effect of picking up on certain kinds of errors in the code.

Reply:
Also see #12028

Reply:
I did have a look at #12028. I think the line previously meant to catch errors related to invalid escape sequences. The 3.8 changelog mentions this - 
``` 
A backslash-character pair that is not a valid escape sequence generates a   
DeprecationWarning since Python 3.6. In Python 3.8 it generates a SyntaxWarning   
instead.(Contributed by Serhiy Storchaka in bpo-32912.)
```
So, I guess it should work. But yeah, needs to be tested more.

Reply:
Tested it. It doesn't. Will look at it once more and try to fix.

Reply:
Tested it, it does flag an error on incorrect escape sequences.
```
*** Error compiling 't.py'...
  File "t.py", line 1
    x = '\e'
        ^
SyntaxError: invalid escape sequence \e
```

Reply:
Looks like this is being removed for the time being, so we need to keep `-We:invalid` for a while https://github.com/python/cpython/pull/15142